lib/utils.ts

```diff
@@ -30,18 +30,26 @@ import * as _ from 'lodash';
  * @example
  * const first = { foo: 'bar' }
  * const second = { foo: 'baz' }
- * const third = { foo: 'qux' }
  *
- * console.log(utils.mergeObjects(first, second, third))
+ * console.log(utils.mergeObjects(first, second))
  * > { foo: 'qux' }
  */
 
-// Notice that this function equals `_.merge` and thus the latter
+// Notice that this function equals `merge` and thus the latter
 // could be used directly, making this function declaration unnecessary.
 // However, we decided to create a new function for this in order to
 // test specific behaviour that affects this module, like function
 // merging.
-export const mergeObjects = _.merge;
+export const mergeObjects = merge;
```

Comment:
I think it'd be much better to make this marginally smarter and avoid all the mess around when trying to use it like:
```suggestion
export const mergeObjects = (target, ...sources) => {
	for (const source of sources) {
		merge(target, source);
	}
	return target;
```

Reply:
I dropped `mergeObjects` in favor of `merge` as Thodoris suggested in another comment